xml/System.Numerics/IFloatingPointIeee754`1.xml

```diff
@@ -143,8 +143,8 @@
       </Parameters>
       <Docs>
         <param name="x">The value to be bitwise decremented.</param>
-        <summary>Decrements a value to the smallest value that compares less than a given value.</summary>
-        <returns>The smallest value that compares less than <paramref name="x" />.</returns>
+        <summary>Decrements a value to the largest value that compares less than a specified value.</summary>
```

Comment:
We should probably make this and `Math`/`MathF` be consistent.

The `returns` wording may be slightly clearer.

Reply:
The `remarks` section is missing for these ones as well, notably.

Reply:
I agree that consistency would be nice. I also like the `Returns the largest value that compares less than a specified value.` phrasing the best.

What do you mean by incomplete, can you elaborate?

Reply:
@tannergooding I updated the wording to be consistent. Let me know if the return section needs any more updates.